nodebuilder/tests/swamp/swamp.go

```diff
@@ -226,6 +227,10 @@ func (s *Swamp) NewNodeWithStore(
 
 	switch t {
 	case node.Bridge:
+		os.Setenv(p2p.EnvKeyCelestiaBootstrapper, "true")
+		defer func() {
+			os.Unsetenv(p2p.EnvKeyCelestiaBootstrapper)
+		}()
```

Comment:
A discussion we had not that far ago(you were probably out) is that our tests should not rely on the presence of a bootstrapper. Also, I'd like to point out that setting this effectively makes all the nodes as bootstrappers because all the nodes share the same ENV space within one test run. 

We set bootstrappers before to enable peer exchange, which is no longer necessary as every FN and BN has peer exchange by default, so in theory, removing this should still make the test work unless I miss anything.

Reply:
Its not only peer exchange, its also the DHT server mode (which is the thing that breaks the tests)

Many swamp tests rely on the presence of the bridge node as a bootstrapper or assume the started bridge node to act as one

Reply:
As discussed in sync, we will go with enabling ServerMode for FN and BN

Reply:
done in #1954, will rebase this pr once merged